feature.md

```diff
@@ -0,0 +1 @@
+# Feature-1
```

Comment:
ya ashmi Great Job..u created pull request to the feature i created :1st_place_medal: